src/hooks/resource.js

```diff
@@ -66,8 +73,17 @@ export function useResourceQuery({ endpointId, filter, datasetId, queryFn, searc
       setSearchValue('')
       setFilters({})
     },
-    refetch() {
-      query.refetch()
+    refetch(value) {
```

Comment:
Why did we change here that much? what's the new usecase

Reply:
Priority is given to the search when searching and filtering RBP.

Reply:
why are clearing search params on refetch. this breaks behavior